gabbi/case.py

```diff
@@ -502,19 +500,14 @@ def _run_request(
         if 'user-agent' not in (key.lower() for key in headers):
             headers['user-agent'] = "gabbi/%s (Python urllib3)" % __version__
 
-        try:
-            response, content = self.http.request(
-                url,
-                method=method,
-                headers=headers,
-                body=body,
-                redirect=redirect,
-                timeout=timeout,
-            )
-        except wsgi_intercept.WSGIAppError as exc:
```

Comment:
Won't removing this break what's described in the docstring above:

```
The call to make the request will catch a WSGIAppError from
wsgi_intercept so that the real traceback from a catastrophic
error in the intercepted app can be examined.
```

See 5b2ac224bdac7a341e461ccaf7c48fb6fa36cfed for when it was added. It's entirely possible that was not right, but it was certainly important at the time.